js/vod-stream-embed.js

```diff
@@ -0,0 +1,17 @@
+// Video-on-demand Stream Video
+
+var ipfs_gateway_self = 'https://ipfs.io'; // IPFS gateway of this node
+var ipfs_gateway_origin = 'https://ipfs.io'; // IPFS gateway of origin stream
+
+var stream_type = 'video/mp4'; // Type of video stream
+var stream_url_ipfs = 'https://ipfs.io/ipfs/' + new URLSearchParams(window.location.search).get('ipfs'); // File path to stream with IPFS content via HTTP server
```

Comment:
@garrying is this a reasonable way of getting a url param? Not sure what input validation may be necessary.

Reply:
I think it's fine. It could be moved to a var to keep the concat succinct.

Reply:
I will wait for full review to make all changes at once.